db/model/model.js

```diff
@@ -44,3 +44,37 @@ exports.fetchArticleByID = (article_id) => {
       throw err;
     });
 };
+
+exports.fetchAllArticles = () => {
+  return (
+    db
+      .query(
+        `SELECT articles.author, articles.title,articles.topic, articles.article_id,articles.topic,
+        articles.created_at, articles.votes, articles.article_img_url,
+        comments.article_id AS comment_count
+        FROM articles 
+        INNER JOIN  
+        comments ON comments.article_id = articles.article_id
+        ORDER BY comments.created_at DESC;`
```

Comment:
We want the articles to be sorted by the article's created_at date not the comment's created_at!